### PR TITLE
JennieB v2.0

### PR DESCRIPTION
**Pull request by jmbcreates**
Something I am trying out.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,7 +18,8 @@ <h2>Advice for Newbies!</h2>
     <section>
       <div class="all-advice-links">
         <div class="skillcrush">
-          <a href="advice/shamiras-advice.html">Shamira's Advice</a><br>
+          <a href="advice/jennieb-advice.html">JennieB's Advice</a><br>
```
Comment (jmbcreates) [inline index.html]: Adding a line at the top.

File: index.html [hunk 2]
```diff
@@ -18,7 +18,8 @@ <h2>Advice for Newbies!</h2>
     <section>
       <div class="all-advice-links">
         <div class="skillcrush">
-          <a href="advice/shamiras-advice.html">Shamira's Advice</a><br>
```
Comment (jmbcreates) [inline index.html]: Bumping this down to the 2nd line

**Comment by redrambles** [comment]
Hey @jmbcreates !

This is really great! The only change we'd like you to make is to rename your css file so that it doesn't start with a '.' - files that start with a '.' are usually reserved for system files.

You can make that change by running:

`git mv css/.jennieb-style.css css/jennieb-style.css`

Then when you run 'git status' - you will see that Git has understood that you want to rename this file. All that's left to do is commit (`git commit -m "your message here"`) and then to push up your branch once more (`git push origin v2.0`) which will automatically update your pull request. 

At that point we should be good to merge in your code! Let us know if you need any help with this.

Cheers,
Ann

**Comment by redrambles** [comment]
Hey @jmbcreates,

We haven't heard from you in a while so I'm going to close this pull request. Feel free to submit a new one when you're ready!

Cheers,
Ann :)

**Pull request closed without merging**